browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -758,6 +763,14 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
       confirmation_phrase);
   html_source->AddString("walletResetTransactionInfoConfirmation",
                          reset_tx_confirmation_text);
+  html_source->AddString(
+      "braveLeoAssistantInputDefaultContextSize",
+      base::NumberToString16(ai_chat::kCustomModelDefaultContextSize));
```

Comment:
nit: move this above the two `html_source` lines, so we only calculate it once

auto default_context_size = base::NumberToString16(ai_chat::kCustomModelDefaultContextSize);
html_source->AddString(
      "braveLeoAssistantInputDefaultContextSize",
      default_context_size);